charts/gateway/README.md

```diff
@@ -61,11 +61,58 @@ Helm Version    Supported Kubernetes Versions
 * [Upgrade the Chart](#upgrading-the-chart)
```

Comment:
For L2, do we need to add Redis?  Is that optional?

Reply:
Also for L2 does mentioning mysql contradict the Important Note below?

Reply:
Added Redis, it is optional